api/app/weather_models/__init__.py

```diff
@@ -1,10 +1,6 @@
 """ Code common to app.weather_models.fetch """
-from datetime import datetime
 from enum import Enum
 import logging
-from scipy.interpolate import interp1d
-
-from app.db.models.weather_models import ModelRunPrediction
 
 logger = logging.getLogger(__name__)
```

Comment:
I think we can get rid of the logging code.

Reply:
Or is it needed because this is __init__.py?

Reply:
Good call, don't think it's needed. We have many `__init__` files with no logger, but logging in individual modules